crates/ika-core/src/dwallet_mpc/crytographic_computation/mpc_computations/dwallet_dkg.rs

```diff
@@ -96,6 +107,23 @@ impl DWalletDKGSecondPartyPublicInputGenerator for DWalletDKGSecondParty {
             VersionedCentralizedDKGPublicOutput::V1(first_round_output) => {
```

Comment:
No but, you should definitely add a `V2` here too - the outputs of this party and the old one are different.
If you want to avoid adding a V2 here, we should have the exact same structure, which I think adds an `SID` there too